tests/Tests/500__jupyterhub/test.robot

```diff
@@ -106,6 +107,17 @@ Start New Pytorch Build
     Wait Until Build Status Is    namespace=redhat-ods-applications    build_name=${new_buildname}   expected_status=Running
     [Return]    ${new_buildname}
 
+Verify Notebook Spawner Modal Does Not Get Stuck When Requesting Too Many Resources To Spawn Server
+   [Documentation]    Try spawning a server size for which there's not enough resources
+   ...    spawner modal should show an error instead of being stuck waiting for resources
+   Select Container Size    X Large
+   Click Button    Start server
+   # This could fail because of https://bugzilla.redhat.com/show_bug.cgi?id=2132043
+   Wait Until Page Contains    Insufficient resources to start    timeout=1min
+   ...    error=Modal did not fail within 1 minute
+   Click Button    Cancel
+   Select Container Size    Small
```

Comment:
why selecting Small? was the goal of the TC to explicitly use a large container?
Plus, if the spawning doesn't fail, this kw would fail and block the overall TC I guess

Reply:
It selects Small because the main test was already using that, so I just reset it back to the original value

This test I don't think will ever fail considering it's selecting XLarge and we never used nodes that are that big

Reply:
hm so why it gets X Large selected?

Reply:
Because I need to make the spawn fail because of "not enough resources", and the easiest way to do that is to try spawning with the biggest size available

Reply:
gotcha